animareflection/ui
# Add `asChild` to Menu trigger component & update static CSS pregeneration

## Description

##### Task link: https://trello.com/c/swa1CQLa/373-update-menu-component-in-ui-library

- Added `asChild` to `Menu` trigger component
- Removed `triggerVariant` prop from `Menu`
- Added env var template with `PREGENERATE_STATIC_CSS=true` for pregenerating static CSS based on this env var
  - Updated Panda config accordingly

~~Note that the deploy builds are failing due to our deploy CI job being rate limited by Vercel. Works if I redeploy from Vercel dashboard, funny enough. Latest: https://ui-storybook-fy1e2cuwf-animareflection.vercel.app~~

## Test Steps

Locally,
1. `cp .env.local.template .env.local` to enable static CSS pregeneration for development
2. In Storybook: verify the `Menu` stories still render properly, including the new `MatchingWidth` story
3. Check for style regressions in example apps (see https://github.com/animareflection/ui/pull/181/files#r1429034304 for details)
